tasks/src/const.ts

```diff
@@ -46,3 +46,93 @@ export const TASKS_MODEL_LIBRARIES: Record<PipelineType, ModelLibraryKey[]> = {
 	"zero-shot-classification":       ["transformers"],
 	"zero-shot-image-classification": [],
 };
+
+/**
+ * Mapping from library name (excluding Transformers) to its supported tasks. 
+ * Inference API should be disabled for all other (library, task) pairs beyond this mapping.
+ * As an exception, we assume Transformers supports all inference tasks.
+ * This mapping is generated automatically by "python-api-export-tasks" action in huggingface/api-inference-community repo upon merge.
+ * Ref: https://github.com/huggingface/api-inference-community/pull/158
+ */
+export const LIBRARY_TASK_MAPPING_EXCLUDING_TRANSFORMERS: Partial<Record<ModelLibraryKey, PipelineType[]>> = {
```

Comment:
can you move it to its own file, i.e. a separate new file, like `./src/lib/interfaces/LibrariesToTasks.ts`?

the `/tasks` folder is more specific to the `/tasks` pages on the Hub, as you mention

Reply:
Cool. Done.